### ISSUE TITLE
`Internal Server Error` on `motion.set_state`

### ISSUE THREAD

**Issue by GabrielInTheWorld** [OpenSlides/openslides-backend#796]
Sending this payload:
```json
[
  {
    "action":"motion.set_state",
    "data":[
      {
        "id":5,
        "state_id":16
      },
      {
        "id":6,
        "state_id":16
      }
    ]
  }
]
```

throws the following error (commit ce01458a9508114c91f4105ab2786f7c3a59c014):
```
backend_1           | DEBUG:urllib3.connectionpool:http://datastore-reader:9010 "POST /internal/datastore/reader/get HTTP/1.1" 200 48
backend_1           | DEBUG:openslides_backend.services.datastore.adapter:Get response with status code 200: {'meta_position': 83, 'next_state_ids': [16, 17, 18]}
backend_1           | [2021-06-22 06:35:08 +0000] [47] [ERROR] Error handling request /system/action/handle_request
backend_1           | Traceback (most recent call last):
backend_1           |   File "/usr/local/lib/python3.8/site-packages/gunicorn/workers/sync.py", line 136, in handle
backend_1           |     self.handle_request(listener, req, client, addr)
backend_1           |   File "/usr/local/lib/python3.8/site-packages/gunicorn/workers/sync.py", line 179, in handle_request
backend_1           |     respiter = self.wsgi(environ, resp.start_response)
backend_1           |   File "/app/openslides_backend/http/application.py", line 121, in __call__
backend_1           |     return self.wsgi_application(environ, start_response)
backend_1           |   File "/app/openslides_backend/http/application.py", line 111, in wsgi_application
backend_1           |     response = self.dispatch_request(request)
backend_1           |   File "/app/openslides_backend/http/application.py", line 41, in dispatch_request
backend_1           |     return self.default_route(request)
backend_1           |   File "/app/openslides_backend/http/application.py", line 68, in default_route
backend_1           |     response_body, access_token = view_instance.dispatch(request)
backend_1           |   File "/app/openslides_backend/http/views.py", line 62, in dispatch
backend_1           |     response = handler.handle_request(request.json, user_id, is_atomic)
backend_1           |   File "/app/openslides_backend/action/action_handler.py", line 98, in handle_request
backend_1           |     results = self.execute_write_requests(self.parse_actions, payload)
backend_1           |   File "/app/openslides_backend/action/action_handler.py", line 131, in execute_write_requests
backend_1           |     write_requests, data = get_write_requests(*args)
backend_1           |   File "/app/openslides_backend/action/action_handler.py", line 168, in parse_actions
backend_1           |     write_request, results = self.perform_action(element, relation_manager)
backend_1           |   File "/app/openslides_backend/action/action_handler.py", line 202, in perform_action
backend_1           |     write_request, results = action.perform(action_data, self.user_id)
backend_1           |   File "/app/openslides_backend/action/action.py", line 145, in perform
backend_1           |     instance = self.base_update_instance(instance)
backend_1           |   File "/app/openslides_backend/action/generics/update.py", line 17, in base_update_instance
backend_1           |     instance = self.update_instance(instance)
backend_1           |   File "/app/openslides_backend/action/actions/motion/set_state.py", line 48, in update_instance
backend_1           |     instance["state_id"] in motion_state["previous_state_ids"]
backend_1           | KeyError: 'previous_state_ids'
```

The start of dispatching the action:
```
backend_1           | [2021-06-22 06:35:08 +0000] [47] [DEBUG] POST /system/action/handle_request
backend_1           | DEBUG:openslides_backend.http.application:Request method is POST.
backend_1           | DEBUG:openslides_backend.http.application:Request contains JSON: [{'action': 'motion.set_state', 'data': [{'id': 5, 'state_id': 16}, {'id': 6, 'state_id': 16}]}].
backend_1           | DEBUG:openslides_backend.http.views:Start dispatching action request.
```

What is wrong there?

**Comment by jsangmeister** [comment]
Not every state has `previous_state_ids` defined: If it's `None`, it should be handled the same as `[]`. I already fixed this in #795, can you check if this problem still occurs?

**Comment by GabrielInTheWorld** [comment]
I think, it is fixed. I will close this issue.
